build/metadata.sh

```diff
@@ -9,6 +9,7 @@ OPERATOR_NAME=${OPERATOR_NAME:-service-telemetry-operator}
 IMAGE_BUILDER=${IMAGE_BUILDER:-podman}
 IMAGE_BUILD_ARGS=${IMAGE_BUILD_ARGS:-''}
 IMAGE_TAG=${IMAGE_TAG:-latest}
+# NOTE(vkmc) Where is this env var being used?
```

Comment:
I mean L13

Reply:
Yup, metadata.sh is used in generate_bundlew

Reply:
Yeah... a quick search though it seems that REQUIRED_OPERATOR_SDK_VERSION is not being used anywhere and should be safe to remove.

Reply:
Yeah... a quick search though it seems that REQUIRED_OPERATOR_SDK_VERSION is not being used anywhere and should be safe to remove.